README.md

```diff
@@ -13,11 +13,15 @@ USAGE:
   ccboc get calculations (Gets all active calculations)
   ccboc --teff=10000 --logG=4.0 get results (Downloads the result of a calculation with teff=10000 and LogG=4.0)
   ccboc get results calc-1881i9dh5zvnllip (Downloads the result of a calculation with id='calc-1881i9dh5zvnllip')
+  ccboc get bulks (Gets all calculation bulks)
+  ccboc get bulk bulk-2bw55pr5p37dasdl (Gets the calculation bulk with id='2bw55pr5p37dasdl')
+  ccboc get workerpools (Gets all the workerpools)
+  ccboc create bulk --bulk-file=<bulk-input-file.json> (Creates a calculation bulk from a file)
       
 
 COMMANDS:
    create, c  Creates a calculation using the values from --teff and --logG flags
-   get, g     Usage: `get calculation CALCID`, or `get calculations` to get all active calculations, or `get results CALCID` to download a calculation's result by CALCID, or `TEFF LOGG get results`.
+   get, g     Usage: `get calculation CALCID`, or `get calculations` to get all active calculations, or `get results CALCID` to download a calculation's result by CALCID, or `TEFF LOGG get results`, or `get bulks` to get all calculation bulks, or `get bulk BULKID` to get a calculation bulk by BULKID., or `get workerpools` to get all workerpools
```

Comment:
This is generally the output of the `--help`. What you are changing here doesn't reflect in the binary itself. Please do the corresponding changes as well, and update the README.md depending on what is the `--help` output.